src/lib.rs

```diff
@@ -28,19 +28,17 @@ mod raw_symbol_token;
 mod raw_symbol_token_ref;
 mod reader;
 mod stream_reader;
+mod symbol;
 mod symbol_table;
 mod system_reader;
 mod writer;
 
-pub use binary::raw_binary_reader::RawBinaryReader;
 pub use data_source::IonDataSource;
-pub use raw_reader::RawStreamItem;
 pub use raw_symbol_token::RawSymbolToken;
-pub use reader::Reader;
-pub use reader::StreamItem;
-pub use stream_reader::StreamReader;
-pub use symbol_table::{Symbol, SymbolTable};
-pub use system_reader::{SystemReader, SystemStreamItem};
+pub use raw_symbol_token_ref::RawSymbolTokenRef;
+
+pub use symbol::Symbol;
```

Comment:
🗺️ `Symbol` has been added as a public module and I reorganized the other exports.